Code/Tools/ProjectManager/Source/GemRepo/GemRepoInspector.cpp

```diff
@@ -86,7 +88,18 @@ namespace O3DE::ProjectManager
         }
 
         // Included Gems
-        m_includedGems->Update(tr("Included Gems"), "", m_model->GetIncludedGemTags(modelIndex));
+        QVector<Tag> includedGemTags;
```

Comment:
If no-one is using the GetIncludedGemTags() method anymore, let's remove it.

Reply:
Updated to use that method instead of doing the work in-line